src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: "What's new in Astro - December 2023"
+description: "December: last month of the year, a time to reflect on all the amazing things we've shipped, including Astro 4.0!"
+publishDate: "December 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-december-2023/og.webp"
+lang: "en"
+---
+
+## Releases
+
+Just in time for the end of the year, we released Astro 4.0!
+
+- [Astro 4.0](https://github.com/withastro/astro/releases/tag/astro%404.0.0)
```

Comment:
We have a shiny blog post we can link to here!

```suggestion
- [Astro 4.0](https://astro.build/blog/astro-4/)
```

Reply:
Shouldn't this be a local link?

```suggestion
- [Astro 4.0](/blog/astro-4/)
```

Reply:
Oh yeah, definitely!